### ISSUE TITLE
box-shadow tool loses ability to modify opacity once it reaches 100%

### ISSUE THREAD

**Issue by argyleink** [GoogleChromeLabs/ProjectVisBug#144]

**Comment by AnnieTaylorCHEN** [comment]
I tried this one on Linuxmint with Chromium. I don't see any change at all. But I'm not sure if I used the right key... I used the Ctrl+left and right arrow.

**Comment by argyleink** [comment]
will try again on chrome on linux and give a report 👍

**Comment by argyleink** [comment]
you are right in reporting a failure here. the metaKey (cmd/windows) dont fire at all, so they're dead keys we cant use. see #194 for a deeper conversation about resolving these keys across platforms. 

otherwise, this issue is still standing which is opacity stops being interactive from the keyboard. todo
